### DISCUSSION TITLE
[Script request] Pinepods

### DISCUSSION THREAD

**Discussion by costa-alex** [community-scripts/ProxmoxVE#996] [Request script]
### Application Name

Pinepods

### Website

https://www.pinepods.online/

### Description

PinePods is a Rust based podcast management system that manages podcasts with multi-user support and relies on a central database with clients to connect to it. It's browser based and your podcasts and settings follow you from device to device due to everything being stored on the server. You can subscribe to podcasts and even hosts for podcasts with the help of the PodPeopleDB. It works on mobile devices and can also sync with a Nextcloud server or gpodder compatible sync server so you can use external apps like Antennapod as well!

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by networkr0** [comment]
I second this request.  It seems like it fill a niche none of the other multimedia streaming apps address.

**Comment by SpaceAgeHero** [comment]
I would also love to have this.
